pandas/core/series.py

```diff
@@ -932,7 +932,7 @@ def _take_with_is_copy(self, indices, axis=0) -> Series:
         """
         return self.take(indices=indices, axis=axis)
 
-    def _ixs(self, i: int, axis: int = 0):
+    def _ixs(self, i: int, axis: int = 0) -> Any:
```

Comment:
Wouldn't mind leaving it un-annotated but I'm also fine using `Any`.

Ideally `Series` becomes generic as done by @Dr-Irv  in the pandas-stubs but that is probably something for the future.